docs/users_guide/landice/test_groups/ismip6_forcing.rst

```diff
@@ -0,0 +1,235 @@
+    # use neareststod method.
+    method_remap = NotAvailable
+
+Below is the example config options that users might create in running
+the test group. This example is for processing the NorESM1-M RCP2.6 repeat
+forcing to the year 2300 onto the 8-80km Antarctic Ice Sheet MALI mesh.
+The example is configured to perform the `Atmosphere.processSMBRacmo` step to
+process the RACMO modern SMB climatology but not the modern thermal forcing.
+
+.. code-block:: cfg
+
+    # config options for ismip6 antarctic ice sheet data set
+    [ismip6_ais]
+
+    # Base path to the input ismip6 ocean and smb forcing files. User has to supply.
+    base_path_ismip6 = /Users/hollyhan/Desktop/Data/ISMIP6-Projections-Forcing-2300/
+
+    # Base path to the the MALI mesh. User has to supply.
+    base_path_mali = /Users/hollyhan/Desktop/Data/MALI-mesh/
+
+    # Forcing end year of the ISMIP6 data. User has to supply.
+    # Available end years are 2100 and 2300.
+    period_endyear = 2300
+
+    # Base path to which output forcing files are saved.
+    output_base_path = /Users/hollyhan/Desktop/Data/ISMIP6-Projections-Forcing-2300_Processed/
+
+    # Name of climate model name used to generate ISMIP6 forcing data. User has to supply.
+    # Available model names are the following: CCSM4, CESM2-WACCM, CSIRO-Mk3-6-0, HadGEM2-ES, NorESM1-M, UKESM1-0-LL
+    model = NorESM1-M
+
+    # Scenarios used by climate model. User has to supply.
+    # Available scenarios are the following: RCP26, RCP26-repeat, RCP85, SSP126, SSP585
+    scenario = RCP26-repeat
+
+    # name of the mali mesh. User has to supply. Note: It is used to name mapping files
+    # (e,g. 'map_ismip6_8km_to_{mali_mesh_name}_{method_remap}.nc').
+    mali_mesh_name = Antarctica_8to80km
+
+    # MALI mesh file to be used to build mapping file (netCDF format). User has to supply.
+    mali_mesh_file = Antarctica_8to80km_20220407.nc
+
+    # config options for ismip6 antarctic ice sheet SMB forcing data test cases
+    [ismip6_ais_atmosphere]
+
+    # Remapping method used in building a mapping file. Options include: bilinear, neareststod, conserve
+    method_remap = bilinear
+
+    # Set True to process RACMO modern climatology
+    process_smb_racmo = True
+
+    # Full path to the RACMO regional climate model SMB data for the modern climatology
+    base_path_racmo = /Users/hollyhan/Desktop/Data/ISMIP6-Projections-Forcing-2300/RACMO2.3p2_ANT27_SMB_yearly_1979_2018/
+
+    # config options for ismip6 ocean thermal forcing data test cases
+    [ismip6_ais_ocean_thermal]
+
+    # Remapping method used in building a mapping file. Options include: bilinear, neareststod, conserve
+    method_remap = bilinear
+
+    # Set to True if the want to process observational thermal forcing data. Set to False if want to process model thermal forcing data.
+    # Note: when set True, the ['ismip6_ais'] config options 'period_endyear', 'model' and 'scenario' will be ignored.
```

Comment:
This comment is a little different from the defaults above.  I think it would be best to make all the comments in the example match the defaults unless there's a reason not to.